src/scripts/eos-analysis

```diff
@@ -663,5 +679,34 @@ def cmd_list_posteriors(args):
         print(name)
 
 
+# Initialize analysis directory
+def cmd_init(args):
```

Comment:
I don't understand where `base_directory` is used

Reply:
Nowhere. When I kept it as an argument, I was considering adding it to the analysis file itself as an override. However, I noticed that this fails when people share a common analysis files across different machines. Hence, I left it for later modification. I can take it out if you like.

Reply:
It has been removed.